diff --git a/src/d3d9/d3d9_common_texture.cpp b/src/d3d9/d3d9_common_texture.cpp
--- a/src/d3d9/d3d9_common_texture.cpp
+++ b/src/d3d9/d3d9_common_texture.cpp
@@ -390,4 +390,7 @@
-    bool isMutable     = m_mapping.FormatSrgb != VK_FORMAT_UNDEFINED;
+    bool isMutable     = m_mapping.FormatSrgb != VK_FORMAT_UNDEFINED;
+#if defined(__ANDROID__)
+    isMutable &= !IsDXTFormat(m_desc.Format);
+#endif
     bool isColorFormat = (formatProperties->aspectMask & VK_IMAGE_ASPECT_COLOR_BIT) != 0;
 
